examples/browser/react-app/package.json

```diff
@@ -13,11 +12,16 @@
     "@types/react-dom": "^18.0.9",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-helmet": "^6.1.0",
+    "react-router-dom": "^6.22.1",
     "react-scripts": "5.0.1",
     "typescript": "^4.6.3",
     "web-vitals": "^3.1.0"
   },
   "scripts": {
+    "buildSDK": "cd ../../../packages/analytics-browser && yarn build ",
+    "installSDK": "cp ../../../packages/analytics-browser/lib/scripts/amplitude-min.js ./public/amplitude.js",
+    "start:dev": "yarn buildSDK && yarn installSDK && yarn start",
```

Comment:
Could you update the readme file to explain how to use these commands?

Reply:
Good call. added the `npm run start:dev` into Readme.